lib/openstack_query/queries/query_wrapper.py

```diff
@@ -17,7 +18,10 @@ class QueryWrapper(QueryMethods, QueryBase):
 
     def __init__(self):
         self._query_results = []
-        self.runner = self.runner_cls(self.prop_enum_cls.get_marker_prop_func())
+        self.executer = QueryExecuter(
+            self.prop_enum_cls,
+            self.runner_cls(self.prop_enum_cls.get_marker_prop_func()),
```

Comment:
Since we already have to prop enum cls we should just do this in the child class